### ISSUE TITLE
Ask for sponsors?

### ISSUE THREAD

**Issue by QuantumInformation** [GaurangTandon/ProKeys#182]
Seems like this extension deserves it.

**Comment by GaurangTandon** [comment]
We appreciate your kind thoughts :)
Although I'd really want to be sure to iron out the minor details still remaining to be in a position to do so.
But before that, what kind of sponsors are you _exactly_ talking about? Patreon based, subscription based featured ads, organization exclusive ProKeys subsets, etc.?

**Comment by QuantumInformation** [comment]
You could try what I'm doing here: https://github.com/QuantumInformation/test-recorder

**Comment by GaurangTandon** [comment]
Interesting approach, I've a question. WRT "please contact me, and I will place your logo here!" on that github page:
I guess - at any given time - there will be many more users viewing and using your app, than the github main page. So, if there are sponsors, why would they place their logo on the github page, instead of inside your app?

**Comment by QuantumInformation** [comment]
Its trivial to add it to the app also, but my audience is purely devs, so there will also be a lot of interest in the repo.

**Comment by GaurangTandon** [comment]
Then it's a good idea :-) My main problem with displaying it inside the app is having frowns from users constantly distracted from ads they never wished to see. Although text based ads might be a good idea.
I'll consult my buddy @iWrote on what he thinks of this idea as well.

**Comment by QuantumInformation** [comment]
Your extension is a productivity boost, don't feel bad about it. You could have a button in the app that opens a window with the sponsor's logos.

**Comment by GaurangTandon** [comment]
Yeah, I could also give my users promo codes to access the sponsor's content at discount rate.

**Comment by QuantumInformation** [comment]
Can I email you?

**Comment by GaurangTandon** [comment]
Sure, I'll receive the email at prokeys.feedback@gmail.com
